src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/rules-engine-edge-app.mdx

```diff
@@ -71,7 +71,7 @@ A inclusão de argumentos em um critério depende dos operadores de comparação
 
 Por exemplo, o seguinte critério identifica se um usuário está usando um navegador web desktop para acessar sua aplicação através de uma expressão regular como argumento:
 
-|  | Variável  | Operador de comparação | Argumento |
+| | Variável | Operador de comparação | Argumento |
```

Comment:
name in column 1

Reply:
Não precisa, é pra renderizar assim

<img width="783" alt="image" src="https://github.com/aziontech/docs/assets/119969414/cfde55f8-ed2a-41fe-a165-6ecfac6468e6">

precisa do If ali porque aparece na UI